testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksChangeDataFeedCompatibility.java

```diff
@@ -416,4 +435,74 @@ public void testDeleteFromNullPartitionWithCdfEnabled()
             onDelta().executeQuery("DROP TABLE IF EXISTS default." + tableName);
         }
     }
+
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_EXCLUDE_73, PROFILE_SPECIFIC_TESTS})
+    public void testThatEnableCdfTablePropertyIsShownForCtasTables()
```

Comment:
IMO, we can move this test to `BaseDeltaLakeMinioConnectorTest`.

Reply:
yes, good point